pkgs/top-level/all-packages.nix

```diff
@@ -27720,6 +27720,8 @@ with pkgs;
 
   smimesign = callPackage ../os-specific/darwin/smimesign { };
 
+  stage0-posix = recurseIntoAttrs (callPackage ../os-specific/linux/stage0-posix { });
```

Comment:
TODO:
- Now that `stage0-posix` is discoverable, do all of the derivations within it have proper meta attributes?
- Are there extraneous attributes?